doc/devel/documenting_mpl.rst

```diff
@@ -820,7 +820,7 @@ example is generated from
     s = 1 + np.sin(2 * np.pi * t)
 
     # Note that using plt.subplots below is equivalent to using
-    # fig = plt.figure and then ax = fig.add_subplot(111)
+    # fig = plt.figure and then ax = fig.add_subplot(1, 1, 1)
```

Comment:
Note that `add_subplot(111)` can be spelt even more succintly as `add_subplot()` now (it defaults to 111).  Especially when you're clearly not going to have a single subplot, I think the argless version is actually the cleanest.

Reply:
agreed.